src/shared/particle_dynamics/continuum_dynamics/continuum_integration.cpp

```diff
@@ -40,7 +40,7 @@ ShearStressRelaxation ::
       velocity_gradient_(particles_->velocity_gradient_), strain_tensor_(particles_->strain_tensor_),
       strain_tensor_rate_(particles_->strain_tensor_rate_), von_mises_stress_(particles_->von_mises_stress_),
       von_mises_strain_(particles_->von_mises_strain_), Vol_(particles_->Vol_),
-      B_(*particles_->getVariableByName<Matd>("KernelCorrectionMatrix")) {}
+      B_(*particles_->getVariableByName<Matd>("FirstConsistencyMatrix")) {}
```

Comment:
If the name changes, might as well call go  with the full name `FirstOrderConsistencyCorrectionMatrix`

Reply:
I have renamed it to LinearGradientCorrectionMatrix for more expressive naming